examples/simple/variables.tf

```diff
@@ -14,10 +14,18 @@
  * limitations under the License.
  */
 
+variable "credentials_file_path" {
```

Comment:
We want to stick to using environment variables rather than explicitly passing credential file paths.